ember/README.md

```diff
@@ -51,19 +51,12 @@ The best place to define the calcite-components is in an initializer.
 ```ember g initializer calcite-components```
 
 ```js
-import {
-  applyPolyfills,
-  defineCustomElements
-} from '@esri/calcite-components/dist/loader';
-
-// Applying polyfills is only necessary if you support IE11/Edge
-applyPolyfills().then(() => {
-  // define calcite components' custom elements on the window
-  // define the resourcesUrl as well
-  defineCustomElements(window, {
-    resourcesUrl: "assets/calcite/"
-  });
+import {defineCustomElements} from '@esri/calcite-components/dist/loader';
 
+// define calcite components' custom elements on the window
+// define the resourcesUrl as well
+defineCustomElements(window, {
+  resourcesUrl: "assets/calcite/"
 });
```

Comment:
You're okay with us using `resourcesUrl` now? I remember you said to use `setAssetPath` since it is documented.

Reply:
I did a quick update, but using `setAssetPath` would be the right choice.